lib/validator.js

```diff
@@ -39,17 +40,20 @@ async function validateExpectations() {
 		}
 	}
 
-	if (req.size !== exp.size) {
```

Comment:
Please don't remove code, but make it work with your use case properly

Reply:
returned old code and tests, now it just excludes optional fields from size check